### PR TITLE
New storage - UI fix for counting documents

### PR DESCRIPTION
**Pull request by berggren**

### CODE REVIEW

File: timesketch/api/v1/resources/sketch.py [hunk 1]
```diff
@@ -370,6 +358,29 @@ def get(self, sketch_id):
                 mapping_dict['type'] = value_dict.get('type', 'n/a')
                 mappings.append(mapping_dict)
 
+        if sketch_indices:
+            # Stats for index. Num docs per shard and size on disk.
+            for timeline in sketch.active_timelines:
+                if indices_metadata[timeline.searchindex.index_name].get('is_legacy', False):
```
Comment (kiddinn) [inline timesketch/api/v1/resources/sketch.py:364]: fix linter...

```
timesketch/api/v1/resources/sketch.py:364:0: C0301: Line too long (93/80) (line-too-long)

```

File: timesketch/api/v1/resources/sketch.py [hunk 2]
```diff
@@ -370,6 +358,29 @@ def get(self, sketch_id):
                 mapping_dict['type'] = value_dict.get('type', 'n/a')
                 mappings.append(mapping_dict)
 
+        if sketch_indices:
+            # Stats for index. Num docs per shard and size on disk.
```
Comment (kiddinn) [inline timesketch/api/v1/resources/sketch.py:362]: not size on disk

File: timesketch/api/v1/resources/sketch.py [hunk 3]
```diff
@@ -370,6 +358,29 @@ def get(self, sketch_id):
...
+            for timeline in sketch.active_timelines:
+                if indices_metadata[timeline.searchindex.index_name].get('is_legacy', False):
+                    doc_count, _ = self.datastore.count(
+                        indices=timeline.searchindex.index_name)
+                    stats_per_timeline[timeline.id] = {
+                        'count': doc_count
+                    }
+                else:
+                    # TODO: Consider using an aggregation here instead?
+                    query_string = f'__ts_timeline_id:{timeline.id}'
```
Comment (kiddinn) [inline timesketch/api/v1/resources/sketch.py:372]: better to use filtering (raw query dsl) or term instead of a query string

**Review by kiddinn — CHANGES_REQUESTED** [review]
first round

File: timesketch/api/v1/resources/sketch.py (hunk 1 above)
Comment (berggren) [reply timesketch/api/v1/resources/sketch.py:364]: Done

File: timesketch/api/v1/resources/sketch.py (hunk 2 above)
Comment (berggren) [reply timesketch/api/v1/resources/sketch.py:362]: done

File: timesketch/api/v1/resources/sketch.py (hunk 3 above)
Comment (berggren) [reply timesketch/api/v1/resources/sketch.py:372]: done

**Review by kiddinn — APPROVED** [review]
LGTM

**Pull request merged**